adi_update_tools.sh

```diff
@@ -146,6 +151,10 @@ rfsom_box ()
 	  cd ./rfsom-box-gui
 	fi
 
+	echo "Repo   : https://github.com/analogdevicesinc/rfsom-box-gui.git" >> $VERSION
+	echo "Branch : master"	>> $VERSION
```

Comment:
Do we want to always take master? I would recommend to turn this into a function where you pass the URL as a parameter. Moreover, if we foresee that we want to save other branches, you might also pass a second parameter with the master name.

3 times doing the same thing is already worth a function...

Reply:
v2: Created a function for writing git repos info in file. Let 'master' as it is for plutosdr_scripts and rfsom repos since they haven't been touched for years.